app/controllers/locations_controller.rb

```diff
@@ -40,12 +40,13 @@ def map_playground
     end
 
     # Show all viewable locations in a demo format
-    field_id = MetadataField.find_by(name: "collection_location").id
+    field_id = MetadataField.find_by(name: "new_collection_location").id
```

Comment:
Is `new_collection_location` a name we keeping long-term? It's a bad practice to name anything "new" because it isn't "new" for long and the old "collection_location" still sounds more authoritative.

Reply:
I made a note to rename to 'collection_location' after the old one is gone but I might as well change to collection_location_v2 now since people keep mentioning it ;) https://czi.quip.com/gzWdAafIX8Yb/LocationMapping-Sprint-Plan

Reply:
added 👍